docs/tutorials/posts/nemotron-nano-v2-evals.md

```diff
@@ -0,0 +1,376 @@
+
+## Prepare evaluation data
+
+We will evaluate the model on the following:
+
+- Science & General reasoning benchmarks:
+    - GPQA
+    - MMLU-Pro
+    - HLE
+
+- Coding reasoning benchmarks
+    - LiveCodeBench
+    - SciCode
+
+- Math reasoning benchmarks:
+    - MATH-500
+    - AIME24
+    - AIME25
+
+- Instruction-following:
+    - IFBench
+
+- Tool-calling:
+    - BFCL v3
+
+- Long-context:
+    - RULER
+
+Here are the commands to prepare these datasets using NeMo-Skills:
+
+```bash
+ns prepare_data gpqa mmlu-pro hle livecodebench scicode bfcl_v3 ifbench math-500 aime24 aime25
+```
+
+For RULER we need to provide extra arguments when preparing the data. If using Slurm cluster,
+make sure to use an appropriate `--cluster` parameter here to ensure the data is being prepared on the cluster itself
+as ruler test files are very large and we want to avoid copying them from your local machine.
+
+```bash
+ns prepare_data --cluster=local ruler \
+    --setup nemotron_nano_128k \
+    --tokenizer_path nvidia/NVIDIA-Nemotron-Nano-9B-v2 \
+    --max_seq_length 131072 \
+    --data_dir /workspace/ns-data
+```
+
+## Evaluation commands
+
+NVIDIA-Nemotron-Nano-9B-v2 is a model which can perform inference in both reasoning and non-reasoning mode.
+We perform evaluations primarily with the reasoning mode. We detail the commands and results for all our evaluations.
+<!-- without specifying any thinking budget, and then we show how to perform evals for `AIME25` with varying thinking budgets. -->
+Note that you might not get exactly the same numbers as reported here because of the stochastic nature of LLM generations.
+
+!!! note
+    The commands provided here assume you're working with a local machine where benchmarks/subsets are evaluated sequentially which will take a very long time. If running on slurm, by default we will run each benchmark and their random seeds as an independent job. You can control the number of parallel
+    jobs with `--num_jobs` parameter.
+
+
+
+### Default Evals
+
+For the reasoning mode evals, we follow the recommended recipe of setting:
+
+- temperature to 0.6
+- top-p to 0.95
+- system_message to '/think'
+- maximum number of generated tokens to 32768
+
+We evaluate all benchmarks in the reasoning on mode, except for RULER, which is evaluated in the reasoning off mode via setting:
+
+- temperature to 0.0
+- top-p to 1.0
+- system message to '/no_think'
+
+
+!!!note
+    The NVIDIA-Nemotron-Nano-9B-v2 is a hybrid model which uses mamba layers along with transformer layers.
+    The model requires the latest version of vllm:
+    ```
+    pip install -U "vllm>=0.10.1"
+    ```
```

Comment:
do we need to add this? Our default container should work, but if you want to keep this, maybe mention that it's only relevant if using non-default container